keyboards/contra/readme.md

```diff
@@ -1,16 +1,24 @@
-CONTRA
-======
+# Contra
 
-![CONTRA](https://cartel.ltd/wp-content/uploads/2018/01/img_3209.jpg)
+![CONTRA](https://raw.githubusercontent.com/ai03-2725/Contra/master/Images/back-2.0.jpg)
```

Comment:
We usually request for images to be hosted on something like imgur

Reply:
The link can be removed if images are mandatory.